tests/AddHttp2ServerPushTest.php

```diff
@@ -93,6 +93,19 @@ public function it_will_not_return_a_push_header_for_inline_js()
         $this->assertFalse($this->isServerPushResponse($response));
     }
 
+    /** @test */
+    public function it_will_return_limit__count_of_links()
```

Comment:
last thing, is there a reason why there is two underscores between `limit__count`?